test/monitoring/components_metrics_test.go

```diff
@@ -24,47 +24,77 @@ import (
 	"testing"
 	"time"
 
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
```

Comment:
internal imports should be after/below external imports. So:
```
v1 "github.com/prometheus/client_golang/api/prometheus/v1"

testutil "github.com/kinvolk/lokomotive/test/components/util"
```

Reply:
done